src/ext/framework/mod.rs

```diff
@@ -59,6 +59,7 @@ mod create_command;
 
 pub use self::command::{Command, CommandType};
 pub use self::configuration::Configuration;
+pub use self::configuration::AccountType;
```

Comment:
Can be simplified to `pub use self::configuration::{AccountType, Configuration};`